HW1/P2/P2b.py

```diff
@@ -1,3 +1,30 @@
+# Your code here
+
 from P2 import *
 
 # Your code here
+
+import findspark
+findspark.init('/home/zelong/spark')
+import pyspark
+import random
+
+sc = pyspark.SparkContext()
+i = range(2000)
+j = range(2000)
+
+random.shuffle(i)
+random.shuffle(j)
+
+rdd_i = sc.parallelize(i, 10)
+rdd_j = sc.parallelize(j, 10)
+
```

Comment:
I hadn't thought about randomizing the data before creating the RDD.  It probably doesn't make a difference here seeing as the data isn't particularly large, but do you have an idea of whether it is faster to randomize pre- or post-RDD?